client/my-sites/plan-features/style.scss

```diff
@@ -1,3 +1,33 @@
+$header-banner-height: 20px;
```

Comment:
Oops, I wasn't sure whether it gets scoped locally or globally if I did that. Good to know, thanks.